documentation/staging/content/release-notes.md

```diff
@@ -38,6 +39,17 @@ draft: false
 
 ### Change log
 
+#### Operator 3.2.2
+
+* Resolved an issue where the operator would retry Kubernetes API requests that timed out without a backoff causing increased network utilization ([#2300](https://github.com/oracle/weblogic-kubernetes-operator/pull/2300)).
+* Resolved an issue where the operator would select the incorrect WebLogic Server port for administrative traffic ([#2301](https://github.com/oracle/weblogic-kubernetes-operator/pull/2301)).
+* Resolved an issue where the operator when running in a large and heavily loaded Kubernetes cluster would not properly detect when a domain had been deleted and recreated ([#2305](https://github.com/oracle/weblogic-kubernetes-operator/pull/2305) and [#2314](https://github.com/oracle/weblogic-kubernetes-operator/pull/2314)).
```

Comment:
operator when running in a large and heavily loaded Kubernetes cluster would -> operator, when running in a large and heavily loaded Kubernetes cluster, would